spanner/spring-data/src/main/com/example/spanner/QuickStartSample.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import com.example.spanner.Singer.Name;
+import java.util.Arrays;
+
+/**
+ * A quick-start sample that uses Spring Data Cloud Spanner to perform read, write, and DDL
+ * operations.
+ */
+public class QuickStartSample {
```

Comment:
I think it's a little unusual that the main class is not the `@SpringBootApplication`.
The three "applications" just seem to be parts of the same one application.

Reply:
as discussed separately, using the component annotation so that the 3 separate files for snippets are separated.